cluster/common.sh

```diff
@@ -71,10 +71,14 @@ function create-kubeconfig() {
   fi
 
   # KUBECONFIG determines the file we write to, but it may not exist yet
-  if [[ ! -e "${KUBECONFIG}" ]]; then
-    mkdir -p $(dirname "${KUBECONFIG}")
-    touch "${KUBECONFIG}"
-  fi
+  IFS=':'
```

Comment:
Make sure you're not mutating this in the global namespace - IIRC you'll need `local IFS` to scope the change to the function only, but it's a little tricky and I'd need to look it up to remember.